### PR TITLE
Gf 1429 guidancepage for pm

### PR DESCRIPTION
**Pull request by venkata999**
Created Guidance page
1. added a 'manage' folder in renderer for all managers views
2. endpoint is given as /manage/opportunities/guidancePM/:opportunityId

### CODE REVIEW

File: src/main/resources/routes [hunk 1]
```diff
@@ -3,14 +3,17 @@
...
-GET         /                                            controllers.StartPageController.startPage
-GET         /opportunities                               controllers.OpportunityController.showOpportunities
-GET         /opportunities/guidance/:opportunityId                      controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
-GET         /opportunity/:opportunityId                  controllers.OpportunityController.showOpportunity(opportunityId:OpportunityId, section:Option[Int])
+GET         /                                                         controllers.StartPageController.startPage
+GET         /opportunities                                            controllers.OpportunityController.showOpportunities
 
-GET         /application_form/:id/apply                  controllers.ApplicationController.showOrCreateForForm(id: ApplicationFormId)
+GET         /opportunities/guidance/:opportunityId                    controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
+GET         /manage/opportunities/guidancePM/:opportunityId           controllers.OpportunityController.showPMGuidancePage(opportunityId:OpportunityId)
```
Comment (jon-shipley) [inline src/main/resources/routes]: I don't like like the "PM" bit of this endpoint in capital letter much.   Would you mind dropping it?

File: src/main/twirl/views/renderers/manage/guidancePM.scala.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@(opportunityId:OpportunityId)
+
+
+@main(s"Guidance on grants - RIFS") {
+<div id="middleframe">
+
+    <div class="gidrow">
+        <div class="column-two-thirds">
```
Comment (jon-shipley) [inline src/main/twirl/views/renderers/manage/guidancePM.scala.html]: We don't need the grid layout for this page, as it doesn't have the RHS column.   You should be able to lose the following divs: middle frame, gidrow, column-two-thirds.

You should add a "text" class to the container enclosing the content to wrap the text at about 8 words in length.

File: src/main/resources/routes (hunk 1 above)
Comment (DougC) [inline src/main/resources/routes]: Agree. Seems redundant since it's under the `/manage` domain.

Not part of this PR, I know, but `application_form` should probably be `application-form` to fit in with url guidelines as well, no?

Comment (venkata999) [inline src/main/resources/routes]: HI thanks. We may see managers other than Portfolio manager in future. We need to distinguish them form each other. Is there any other  name you suggest

Comment (DougC) [inline src/main/resources/routes]: Nope. Let's deal with that when we get there.

**Comment by DougC** [comment]
Yep, `/manage/opportunity/:id/guidance` would make sense

File: src/main/resources/routes [hunk 3]
```diff
@@ -33,6 +35,9 @@ GET         /application/:id/section/:num/item/:itemNum/edit          controller
 GET         /reset                                                    controllers.ApplicationController.reset
 GET         /wip                                                      controllers.OpportunityController.wip(backUrl: String)
 
+# Manage Views
+GET         /manage/opportunities/guidance/:opportunityId           controllers.OpportunityController.showPMGuidancePage(opportunityId:OpportunityId)
```
Comment (DougC) [inline src/main/resources/routes]: I don't think it makes sense to have the `opportunityId` as part of the url path. Isn't the text of the guidance page the same for all opportunities?

I know that the guidance template needs to generate a link back to the opportunity detail, but we could pass it in as a query parameter rather than a path segment. If you change this to `/manage/opportunities/guidance` then the play parameter to the controller as a query parameter.
